### PR TITLE
Add Slasher and Disputer docs

### PR DESCRIPTION
**Pull request by arajasek**
As in the title. 

Work by @TippyFlitsUK, only opening the PR to leave review.

### CODE REVIEW

File: content/en/lotus/manage/slasher-and-disputer.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+# The Lotus Slasher & Lotus Disputer
+
+Lotus Slasher and Lotus Disputer are essential services provided by the Filecoin protocol. They help maintain network integrity, prevent bad actors, and reward active participants. 
+
+The Slasher and Disputer services are designed to be highly efficient and lightweight processes that require minimal resources. Enabling these services will have no detrimental effect on the performance of your Lotus node.
+
+# The Lotus Slasher
+
+The Filecoin protocol enforces penalties for three types of consensus faults that can be committed by block producers. To detect and report these faults, any user running a Lotus node can leverage the Lotus Slasher service and invoke the `ReportConsensusFault` method on the respective miner actor. Detailed information about these three consensus faults can be found [here](https://github.com/filecoin-project/lotus/blob/dbbcf4b2ee9626796e23a096c66e67ff350810e4/chain/vm/fvm.go#L113-L132).
```
Comment (arajasek) [inline content/en/lotus/manage/slasher-and-disputer.md:23]:
```suggestion
The Filecoin protocol enforces penalties for three types of Consensus Faults that can be committed by block producers. To detect and report these faults, any user running a Lotus node can leverage the Lotus Slasher service. This service monitors all incoming blocks for potential Consensus Faults, and invokes the `ReportConsensusFault` method on the respective miner actor. Detailed information about these three Consensus Faults can be found [here](https://github.com/filecoin-project/lotus/blob/dbbcf4b2ee9626796e23a096c66e67ff350810e4/chain/vm/fvm.go#L113-L132).
```

File: content/en/lotus/manage/slasher-and-disputer.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+When a consensus fault is successfully identified by the Lotus Slasher, the block producer responsible for the fault is subjected to a fine equivalent to 5 times the current epoch's block reward value. Additionally, they are ineligible to produce any further blocks for a full finality period of 900 epochs. Furthermore, the offending block producer is prohibited from pre-committing or recovering storage during the same finality period.
```
Comment (arajasek) [inline content/en/lotus/manage/slasher-and-disputer.md:25]: Totally didn't know you couldn't recover storage when under a fault! Thanks!

File: content/en/lotus/manage/slasher-and-disputer.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+
+The Lotus Slasher functionality is activated by simply providing additional flags to the `lotus daemon` launch command as shown below.
+
+```shell
+lotus daemon --slash-consensus=true --slashdb-dir="/your/path"
+```
+
+The `--slash-consensus` and `--slashdb-dir` flags are both required to succesfully invoke the Slasher service.
+
+The `--slasher-sender` flag is also provided to optionally set the account to report consensus faults from.
```
Comment (arajasek) [inline content/en/lotus/manage/slasher-and-disputer.md:41]: Can we add that if this isn't provided, the wallet's default address will be used?

File: content/en/lotus/manage/slasher-and-disputer.md [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+
+
+# The Lotus Disputer
+
+Prior to the introduction of [FIP-0010](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0010.md), Storage Providers were required to verify and record proofs of continued data storage in the form of on-chain WindowPoST messages. This approach often led to high costs and bandwidth usage associated with verifying proofs, especially during periods of congestion and rising gas fees.
```
Comment (arajasek) [inline content/en/lotus/manage/slasher-and-disputer.md:46]: This is just opinion, but I don't think this section needs to be written with so much history -- the motivation behind FIP-0010 doesn't seem too relevant here.

I'd consider rewriting this to just say that:
- WindowPoSts are important proofs submitted daily to ensure continued storage of all data on Filecoin
- These proofs are NOT actually validated on chain, in order to save chain bandwidth
- Instead, the network relies on off-chain validators to actually check the correctness of these proofs, and `Dispute` them if appropriate

And then everything you have from `Third-party Lotus node operators now...` onwards looks good!

File: content/en/lotus/manage/slasher-and-disputer.md [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+   --max-fee value  Spend up to X FIL per DisputeWindowedPoSt message
+   --from value     optionally specify the account to send messages from
+```
+
+## Run the Disputer
+
+Simply run `lotus chain disputer start` to start the disputer. The Disputer will automatically begin to dispute proofs from the current epoch unless a specific height is set using the `--start-epoch` flag.
+
+{{< alert icon="tip" >}}
+The Lotsu Disputer will take a couple of minutes to fully start. Any `rpc output message buffer` warning messages displayed during the the initiation process can be safely ignored.
```
Comment (arajasek) [inline content/en/lotus/manage/slasher-and-disputer.md:75]:
```suggestion
The Lotus Disputer will take a couple of minutes to fully start. Any `rpc output message buffer` warning messages displayed during the the initiation process can be safely ignored.
```

File: content/en/lotus/manage/slasher-and-disputer.md [hunk 6]
```diff
@@ -0,0 +1,84 @@
...
+{{< /alert >}}
+
+## Manual Disputes
```
Comment (arajasek) [inline content/en/lotus/manage/slasher-and-disputer.md:78]: There's nothing incorrect here, but I'd propose dropping this section -- there's no real situation when a user will want to do this.

Honestly, we might want to hide this command entirely, it only really exists for manual testing.

**Review by arajasek — COMMENTED** [review]
Generally looks great, everything seems correct to me! I have a few suggestions, but they're all optional.

File: content/en/lotus/manage/slasher-and-disputer.md (hunk 3 above)
Comment (TippyFlitsUK) [inline content/en/lotus/manage/slasher-and-disputer.md:41]:
```suggestion
The `--slasher-sender` flag is also provided to optionally set the account to report consensus faults from. If this flag is not set, the default wallet address will be used.
```

Comment (TippyFlitsUK) [reply content/en/lotus/manage/slasher-and-disputer.md:41]: Good call!!

**Review by TippyFlitsUK — COMMENTED** [review]
Minor updates

File: content/en/lotus/manage/slasher-and-disputer.md (hunk 6 above)
Comment (TippyFlitsUK) [reply content/en/lotus/manage/slasher-and-disputer.md]: Agreed!! I can't think of any situation where this would be used!

**Review by TippyFlitsUK — APPROVED** [review]

**Pull request merged**